changelog/29060.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
```

Comment:
I think `improvement` is more appropriate for this, we reserve `feature` for "large topical additions for a major release"